app/scripts/modules/ecs/serverGroup/configure/wizard/verticalScaling/verticalScaling.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="container-fluid form-horizontal">
+
+  <div class="form-group">
+    <div class="col-md-5 sm-label-right">
+      Reserved compute units <help-field key="ecs.capacity.reserved.computeUnits"></help-field>
+    </div>
+    <div class="col-md-2"><input type="number"
+                                 class="form-control input-sm no-spel"
+                                 ng-pattern="basicSettingsCtrl.detailPattern"
+                                 name="computeUnits"
+                                 ng-model="$ctrl.command.computeUnits"/></div>
+  </div>
+
+  <div class="form-group">
+    <div class="col-md-5 sm-label-right">
+      Reserved Memory <help-field key="ecs.capacity.reserved.memory"></help-field>
```

Comment:
Inconsistent casing - `Memory` should be `memory`, or `compute units` above should be `Compute Units`

Reply:
Made it `Compute Units`